### ISSUE TITLE
Using a lower bound for version pinning with alpine apk triggers version pinning warnings and stdout redirection error

### ISSUE THREAD

**Issue by stonecharioteer** [hadolint/hadolint#782]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

This should not have had any errors

### Actual behavior
Using `>=` in version pinning triggers 2 events: hadolint thinks that the version isn't pinned since it uses `>=` instead of `=`, and it also thinks that the STDOUT is redirected twice.

### Steps to reproduce the behavior

I'm using the following in an alpine image.

```Dockerfile
FROM rust:1.59-alpine3.14 AS builder
WORKDIR /app
RUN apk add --no-cache musl-dev>=1.1 pkgconfig>=0.23 openssl>=1.1 libressl-dev>=3.5
```

hadolint version is v2.8.0.

```bash
Dockerfile:3 DL3018 warning: Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`
Dockerfile:3 SC2261 error: Multiple redirections compete for stdout. Use cat, tee, or pass filenames instead.
```

**Comment by lorenzo** [comment]
Would you like to contribute a fix for this? It should be simple to do and I can guide you

**Comment by stonecharioteer** [comment]
@lorenzo Yes! I'd love to. How do I get started?

**Comment by stonecharioteer** [comment]
While the [official documentation doesn't specifically show `>=`](https://wiki.alpinelinux.org/wiki/Package_management#Holding_a_specific_package_back), here's a screenshot of me trying it out in an alpine container. 

![2022-03-31-151921_772x407_scrot](https://user-images.githubusercontent.com/11478411/161028172-4d2a9ca7-b286-4972-a3f2-a74282c72742.png)

**Comment by lorenzo** [comment]
@[stonecharioteer](https://github.com/stonecharioteer) Thanks, let's focus first on the first warning. You will need to do this:

* You need to edit this file https://github.com/hadolint/hadolint/blob/master/src/Hadolint/Rule/DL3018.hs#L32
* Repeat the same line I linked to, so that it contains `>=` instead of `=`
* Add a test, like this at the bottom of the file https://github.com/hadolint/hadolint/blob/master/test/Hadolint/Rule/DL3018Spec.hs#L28-L38



Thanks for your help

**Comment by m-ildefons** [comment]
Fixed in v2.15.0
